### DISCUSSION TITLE
[Spaceship] [Appstore Connect] Modify the approved in-App purchases

### DISCUSSION THREAD

**Discussion by 954708775** [fastlane/fastlane#17284] [Q&A] [answered]
Hello Everyone!
I have checked the related issues but unfortunately found that it seems couldn't modify the approved in-App purchase via spaceship. So I just want to confirm it again.

It's my first time to share my opinion in Git-hub, hope these issues could be figured out.

Thanks.

**Comment by 954708775** [comment, marked as answer]
Spaceship really helps me a lot, but I wonder how could I change the existing description or add new localizations.

**Reply by max-ott** [reply]
Should still work pretty much the same as in https://github.com/fastlane/fastlane/pull/7834 The `ConnectAPI` changes from Apple have not arrived at IAPs (yet).

**Reply by 954708775** [reply]
Thanks, max-ott, it works.
